.github/dependabot.yml

```diff
@@ -8,10 +8,3 @@ updates:
     versioning-strategy: "auto"
     allow:
       - dependency-name: "@seamapi/*"
-      - dependency-type: production
-    groups:
```

Comment:
Can we still use a group so we get one pr for all the `@seamapi/*` deps?

Reply:
This should work